### ISSUE TITLE
Expose FilterStates objects out of teal.slice

### ISSUE THREAD

**Issue by nikolas-burkoff** [insightsengineering/teal.slice#102]
See https://github.com/insightsengineering/teal.slice/pull/100#issuecomment-1255242315 and here https://github.com/insightsengineering/teal.slice/pull/100#issuecomment-1255872787

**Comment by donyunardi** [comment]
I don't think this is relevant anymore with implementation of teal_slices() and teal_slice()

**Comment by donyunardi** [comment]
@gogonzo please confirm and close.

**Comment by chlebowa** [comment]
The underlying problem persists and will only go away with [this](https://github.com/insightsengineering/teal.slice/pull/498).
